zerver/lib/bugdown/__init__.py

```diff
@@ -754,11 +754,14 @@ def handleMatch(self, match):
 def make_emoji(codepoint, display_string):
     # type: (Text, Text) -> Element
     src = '/static/generated/emoji/images/emoji/unicode/%s.png' % (codepoint,)
+    # Replace underscore in emoji's title with space
+    title = display_string[1:-1].replace("_", " ")
+
     elt = markdown.util.etree.Element('img')
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
```

Comment:
This is basically totally unrelated to this PR, but reading this got me asking: Does it actually make sense to set both `title` and `alt`?  It seems like we could just have one of the two, and have it be the value we have as "title" in this function.

@vaidap thoughts on the screenreader angle here?